sdk/storage/azure-storage-blob/azure/storage/blob/_shared_access_signature.py

```diff
@@ -47,7 +46,7 @@ def __init__(self, account_name, account_key=None, user_delegation_key=None):
             A user delegation key can be obtained from the service by authenticating with an AAD identity;
             this can be accomplished by calling get_user_delegation_key on any Blob service object.
         '''
-        super(BlobSharedAccessSignature, self).__init__(account_name, account_key, x_ms_version=X_MS_VERSION)
+        super(BlobSharedAccessSignature, self).__init__(account_name, account_key, **kwargs)
```

Comment:
probably we want to modify doc for x_ms_version in this class? it's not a param now